x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/utils/persistence/ResultsPersisterService.java

```diff
@@ -148,12 +180,28 @@ public SearchResponse searchWithRetry(SearchRequest searchRequest,
             } catch (ElasticsearchException e) {
                 LOGGER.warn("[" + jobId + "] Exception while executing search action", e);
                 failureMessage = e.getDetailedMessage();
+                if (isIrrecoverable(e)) {
+                    LOGGER.warn(new ParameterizedMessage("[{}] experienced irrecoverable failure", jobId), e);
```

Comment:
Could you make this message consistent with others? "...failure that cannot be..."